chapters/stress/index.md

```diff
@@ -4,33 +4,62 @@
+This example benchmark prioritizes a real-world scenario, where a user hosts Jina in the cloud. We use six machines in AWS, of instace size `c5.2xlarge` (8 vCPUs, 16GB RAM). We also add 100GB gp2 ssd.
 
-### Datasets
+The instances are split as follows:
 
-To run these tests for Jina, we have prepared 2 scenarios:
+- **Client**: from this machine we create the Flows and issue the client requests (indexing and querying)
+[comment]: <> (- Crafter: this machine TODO is this used?)
+- **Flow**: this machine hosts the `Gateway` to the `Flow`
+- **Encoder**: this machine processes the raw data (text or image) with the respective `Encoders`
+- **Indexer**: this machine hosts the `Indexers`
+- **Ranker**: this machine hosts the `Redis` servers and the `Rankers`
 
-#### Image
+## Datasets
 
-The first scenario simulates an image search application. Images are randomly generated and indexed using the flow with this
-default structure.
+To run these tests, we prepared two scenarios: text and image search. These are the most common use cases for neural search. 
+
+Note that in these tests, the quality of the results is not assessed, and therefore only the amount of returned matches is validated.
+
+### Image search
+
+The first scenario simulates an image search application. Images are randomly generated and indexed using a Flow with this topology:
 
 ![Image Index Flow](image_index_flow.png)
 
-At query time, the Flow looks like this one.
+We use the `mobilenet_v2` `Encoder` with the `ImageTorchEncoder`.
+
+At query time, the Flow looks like this:
 
 ![Image Query Flow](image_query_flow.png)
 
-In the Image Search scenario, there is no `Segmentation` involved and no `Reranking` of the results is used.
-The results are returned according to the `distance` in embedding space returned by the `VectorIndexers`. In these tests,
-no quality is assessed and therefore only the amount of returned matches is validated.
+In the image search scenario, there is no `Segmentation` and ranking. The results are returned according to the `distance` in embedding space returned by the `VectorIndexers`. 
+
+### Text search
+
+The text search topology is similar to the one above, but with a `Segmenter` (`Sentencizer`) and a `Ranker` (`SimpleAggregateRanker`). This means we are indexing on the chunk level of a `Document`. 
+
+The full indexing Flow can be seen below:
+
+[comment]: <> (TODO)
 
-###Configuration
+The full search Flow can be seen below:
 
-The configuration parameters of this experiment can be found in the `.env` file, whose values are set as environment variables
-and changed in the `flow` and `pod` yamls.
+[comment]: <> (TODO)
+
+## Configuration of Flows and Pods
+
+Jina allows for parallelization of data processing. You can read an overview of these options [here](link)
+
+[comment]: <> (TODO link)
+
+We use these to parallelize the various `Pods`. We also use the `load_balance` scheduling strategy. This prioritizes the idle `Peas` inside the `Pods`.    
```

Comment:
I would not say that we do load balancing, but say that one can parametrize the scheduling strategy